_posts/2015-12-18-Rails-5-0-beta1.markdown

```diff
@@ -0,0 +1,78 @@
+
+There's still more work to be done on this feature, but we're off to a great start. By default, API mode just relies on #to_json calls on model classes. But you can either use [Jbuilder][jbuilder], [Active Model Serializers][ams], or look at the new [JSONAPI::Resources][jsonapi] project for a more advanced solution.
+
+Thanks in particular to [Santiago Pastorino][santiagopastorino] and [Jorge Bejar][jorgebejar] for making this happen.
+
+## One Rails command to rule them all
+Why are some commands living in bin/rails and some commands living in bin/rake? That's a common question, especially for beginners, and we never had a good answer (just lots of technical excuses). So now we've committed to making bin/rails the one master command to rule them all. All your rake commands are available through here as a gateway, but we'll eventually port many of them over. So your fingers will now have to get used to "rails db:migrate" instead of "rake db:migrate". That should only take a few months!
+
+[Kasper Timm Hansen][kasper] has been herding this project.
+
+## A few other highlights
+
+* ApplicationRecord has been born as a default parent class of all models created by the generators
+* ActiveRecord::Relation#in_batches makes it much easier to deal with record work in batches at the time to lessen memory overloads.
+* Post.where('id = 1').or(Post.where('id = 2')) gives you exactly what you'd think!
+* No more accidentally halting Active Record callbacks because the last statement is false. Now you throw(:abort) explicitly!
+
+You should really checkout the CHANGELOGs, though. There's just so much new and good stuff available in all the frameworks:
+
+* [Action Mailer CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionmailer/CHANGELOG.md)
+* [Action Pack CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionpack/CHANGELOG.md)
+* [Action View CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionview/CHANGELOG.md)
+* [Active Model CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activemodel/CHANGELOG.md)
+* [Active Record CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activerecord/CHANGELOG.md)
+* [Active Support CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activesupport/CHANGELOG.md)
+* [Active Job CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activejob/CHANGELOG.md)
+* [Railties CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/railties/CHANGELOG.md)
+
+## Maintenance consequences
+
+As per our [maintenance policy][maintenance-policy], the release of Rails 5.0 will mean that bug fixes will only apply to 5-0-stable, regular security issues to 5.0.x, 4.2.x, and severe security issues to 5.0.x, 4.2.x, and 4.1.x. Ruby 2.2+ is now also the only supported version of Ruby for Rails.
```

Comment:
> Ruby 2.2+ is now also the only supported version of Ruby for Rails.

...is slightly confusing, because

1. Only 5.0 requires 2.2+, not the other versions mentioned in the last sentence;
2. 2.2+ is technically not a single "version" (singular). In particular, we don't want to confuse people into thinking that we don't support 2.3

"Rails 5.0 also requires Ruby 2.2 and above."?